doc/scapy/layers/bluetooth.rst

```diff
@@ -71,12 +71,28 @@ There are multiple protocols available for Bluetooth through ``AF_BLUETOOTH``
 sockets:
 
 Host-controller interface (HCI) ``BTPROTO_HCI``
-  Scapy class: ``BluetoothHCISocket``
```

Comment:
Leave this? and eventually move your new description in this?
I'd also add that `BluetoothMonitorSocket` and `BluetoothUserSocket` are essentially subtypes of HCI sockets (if I understand this correctly).

Reply:
Sorry for the delay, I've been AFK for a couple of days...

I wanted to provide a generic description for the BPROTO_HCI interface so I moved BluetoothHCISocket down...
I had my doubts on how to order and clasify the sockets as BluetoothMonitorSocket is not purely HCI but finally, I've decided to document from more generic towards more specific and decided that the BluetoothUserSocket should go first as it is the "recommended" one...

Do you still believe I should leave BluetoothHCISocket first? Thanks!

Reply:
Sorry I might be confusing, I can't really visualise REST in my head.

Does the doc look formatted like this:

- HCI
    - monitor
    - User
    - HCI
- L2PCAP

If so it's all good.

Reply:
That is correct! Resolving! :D